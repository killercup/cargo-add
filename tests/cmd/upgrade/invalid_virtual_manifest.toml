bin.name = "cargo-upgrade"
args = ["upgrade", "--manifest-path", "Cargo.toml"]
status.code = 1
fs.sandbox = true

[env.add]
CARGO_IS_TEST="1"
