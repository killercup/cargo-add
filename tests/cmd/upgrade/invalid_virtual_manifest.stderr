Command failed due to unhandled error: Found virtual manifest, but this command requires running against an actual package in this workspace. Try adding `--workspace`.

